src/polyglot-notebooks-vscode-common/src/variableExplorer.ts

```diff
@@ -150,7 +150,21 @@ class WatchWindowTableViewProvider implements vscode.WebviewViewProvider {
 
     private setRows(rows: VariableGridRow[]) {
         if (this.webview) {
-            this.webview.postMessage({ command: 'set-rows', rows });
+
+            const localizationStrings: GridLocalization = {
+                actionsColumnHeader: 'Actions',
+                nameColumnHeader: 'Name',
+                valueColumnHeader: 'Value',
+                typeColumnHeader: 'Type',
+                kernelNameColumnHeader: 'Kernel',
+                shareTemplate: 'Share value {value-name} from {kernel-name} kernel'
+            };
+
+            this.webview.postMessage({
+                command: 'set-rows',
+                rows,
+                localizationStrings: localizationStrings
```

Comment:
I noticed that the messages to set rows are sending the column names as well. I was wondering if it would be possible to set the column names only once before sending the rows. It seems like the column names won't change often, so is there really a need to keep them in the state?